frontend/public/components/nav/nav-header.tsx

```diff
@@ -53,25 +64,46 @@ const NavHeader: React.FC<NavHeaderProps> = ({ onPerspectiveSelected }) => {
     [isPerspectiveDropdownOpen, togglePerspectiveOpen],
   );
 
-  const perspectiveItems = React.useMemo(
-    () =>
-      perspectiveExtensions.map((nextPerspective: Perspective) => (
+  const perspectiveItems = React.useMemo(() => {
+    const items = perspectiveExtensions.map((nextPerspective: Perspective) => (
+      <DropdownItem
+        key={nextPerspective.properties.id}
+        onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
+          onPerspectiveSelect(event, nextPerspective)
+        }
+        isHovered={nextPerspective.properties.id === activePerspective}
+      >
+        <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
+          <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
+          {nextPerspective.properties.name}
+        </Title>
+      </DropdownItem>
+    ));
+    if (acmLink) {
+      items.push(
         <DropdownItem
-          key={nextPerspective.properties.id}
-          onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
-            onPerspectiveSelect(event, nextPerspective)
-          }
-          isHovered={nextPerspective.properties.id === activePerspective}
-          component="button"
+          key={ACM_LINK_ID}
+          onClick={() => {
+            window.location.href = acmLink.spec.href;
+          }}
+          isHovered={ACM_LINK_ID === activePerspective}
         >
           <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
-            <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
-            {nextPerspective.properties.name}
+            <span className="oc-nav-header__icon">
+              <img
+                src={acmIcon}
+                height="12em"
+                width="12em"
+                alt="Advanced Cluster Management icon"
```

Comment:
No need for alt text here since it's decorative. `alt=""` is acceptable in that case.

```suggestion
                alt=""
```

Reply:
fixed